README.md

```diff
@@ -1,2 +1,98 @@
-# passports-stubber
-Create stub API services using a simple JSON configuration
+# hmpo-stubber
+Create dynamic stub API services using a simple JSON configuration
+
+## Usage
+```
+const stubber = require('hmpo-stubber');
+
+app.use('/stubs', stubber.middleware('./services.json', 'My Stub Server'));
+```
+
+stubber.middleware(services, [name], [basePath]);
+    services: Array, // array of Service config objects
+        (can be a filename, each service can be a filename)
+    name: String(Stub), // name of stub server for debug logging
+    basePath: String(./), // relative location when looking up filenames
+
+## Command line interface
+```
+hmpo-stubber [options] [services.json...]
+
+  -c, --config path::String  Use configuration from this file
+  -p, --port Number          Specify port to listen on - default: 3030
+  -m, --mount String         Base path to mount mocks on - default: /
+  -s, --scenario String      Specify the default scenario
+  -h, --help                 display help
+```
+
+### Command line config file
+    {
+        port: Number(3030), // port number
+        mount: String('/'), // base URL to mount the standalone server
+        scenario: String(), // default scenario
+        stubs: Array,       // array of service filenames or service lists, 
+    }
+
+## Configuration
+
+### Service config object
+    {
+        name: String,        // service name for debug logging
+        method: String(GET), // method to mount in the stub server
```

Comment:
Our services are currently defined as `://host/end-point` without any method.

This does mean that there is a chance that an endpoint could expose different but working responses on GET and POST. Also, if we have any properly RESTy services, they'll also expose PUT and DELETE.

Question becomes, would we then have 4 different stubbed service configs for 1 app configured service?